Mathlib/RingTheory/MatrixAlgebra.lean

```diff
@@ -16,8 +16,6 @@ universe u v w
 
 open TensorProduct
 
-open TensorProduct
```

Comment:
You could also combine these lines. (If so, can you check if all of these `open`s are necessary?)